### ISSUE TITLE
some problems sending over the GELI key to the target

### ISSUE THREAD

**Issue by Phorze** [hughobrien/zfs-remote-mirror#8]
First of All, great guide, been waiting for something like this for an offsite backup.

When i try to push the geli key using the fifo key with your command it seems the contents of the key are literally interpreted on the receiving end. resulting somehow in the cat command never executing, but ending with an error like this:

`
user@host:~# ssh backup-knox-fifo < ~/.ssh/knox-geli-key &
Pseudo-terminal will not be allocated because stdin is not a terminal.
Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
Unmatched '"'.
`
Any pointers on how to fix this? Thanks.

**Comment by hughobrien** [comment]
did you setup the `backup-knox-fifo` key to trigger the correct command?

`force-command="mkfifo -m 600 /tmp/k; cat - > /tmp/k; rm -P /tmp/k"`

The call to `cat` should trigger the command to read from `stdin`

**Comment by Phorze** [comment]
Yes i did. will try to regen the keys as soon as i am able,  to see if i made an error. Then i will report back

**Comment by Phorze** [comment]
Regenerated the keys, but the same problem occurs.
When i replace the key contents with a simple string the following error comes back:
teststring: Command not found.

Might it be the wrong shell type on the knox side? 

user@rpi:~ # echo $shell
/bin/csh
